proto/cosmos/group/v1/tx.proto

```diff
@@ -342,8 +348,8 @@ message MsgExec {
   // proposal is the unique ID of the proposal.
   uint64 proposal_id = 1;
 
-  // signer is the account address used to execute the proposal.
-  string signer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // address is the account address used to execute the proposal.
+  string address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
This doesn't seem more clear nor similar to how we normally use address fields

Reply:
> nor similar to how we normally use address fields

We already use `address` as signer field multiple times in x/group, see https://github.com/cosmos/cosmos-sdk/pull/11515#discussion_r839626215. Initial rationale was consistency.

I renamed to `executor` here, and did some other `addressz renames in b40cd8adff68b4a6c33acbd3a658a5fb080a642b. LMK if you can think of better names.